client/src/components/helpers/avatar-renderer.tsx

```diff
@@ -26,9 +27,13 @@ function AvatarRenderer({
 
   useEffect(() => {
     const validationImage = document.createElement('img');
-    validationImage.src = picture;
-    validationImage.onload = onImageLoad;
-    validationImage.onerror = onImageError;
+    if (isURL(picture)) {
```

Comment:
Do we not want to include some of the options like `{require_protocol: true}`

Reply:
It's not strictly necessary since any invalid url should result in an error when we `src` it.  We can, though. 

Which ones did you have in mind?

Reply:
As far as I can see, that is the only default one I would change. I forgot about our sourcing tests, though. So, it is fine either way.